base/src/com/google/idea/blaze/base/wizard2/ui/BlazeEditProjectViewControl.java

```diff
@@ -600,9 +600,6 @@ public void updateBuilder(BlazeNewProjectBuilder builder) {
               .add(
                   ScalarSection.builder(ImportSection.KEY)
                       .set(selectProjectViewOption.getSharedProjectView()))
-              .add(
-                  ScalarSection.builder(TryImportSection.KEY)
```

Comment:
Please remove the `TryImportSection` import.

Reply:
done

Reply:
It will be not wrong to ask ErrorProne to flag unused imports and fail the build... ;)